mosaic/queries/ThroughputQuery.py

```diff
@@ -11,10 +11,6 @@ class ThroughputQuery(BaseQuery):
                        'DURING("{begin_date}", "{end_date}")')
     }
 
-    def set_defaults(self):
```

Comment:
One thing to note: this set_defaults pattern of setting the types probably exists across several (if not all) queries. This was probably an oversight on my end that should be refactored. Removing it here makes sense since the value will be passed in from the args now, we probably should just go ahead and remove it everywhere.

Reply:
Yes, I would agree.